wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+
+## Article naming
+
+*See also: [Folder names](#folder-names) and [Titles](#titles)*
+
+Article titles should be singular and use sentence case. See [Wikipedia's naming conventions article](https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(plurals) "Wikipedia") for more details.
+
+Article titles should match the folder name it is in (spaces may replace underscores (`_`) where appropriate). If the folder name changes, the article title should be changed to match it and vice versa.
+
+---
+
+Contest and tournament articles are an exception. The folder name must use abbreviations, acronyms, or initialisms. The article's title must be the full name of the contest or tournament.
+
+## Folder and file structure
+
+### Folder names
+
+*See also: [Article naming](#article-naming)*
+
+Folder names must be in English and use sentence case.
+
+Folder names must only use these characters:
+
+- uppercase and lowercase letters
+- numbers
+- underscores (`_`)
+- hyphens (`-`)
+- exclamation marks (`!`)
+
+### Article file names
+
+The file name of an article can be found in the `File Name` column of the [locales section](#locales). The location of a translated article must be placed in the same folder as the English article.
+
+### Index articles
+
+An index article must be created if the folder is intended to only hold other articles. Index articles must contain a list of articles that are inside its own folder. They may also contain other information, such as a lead paragraph or descriptions of the linked articles.
+
+### Disambiguation articles
+
+[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder. The main page must be updated to include the disambiguation article. Refer to [Disambiguation/Mod](/wiki/Disambiguation/Mod) as an example.
+
+Redirects must be updated to have the ambiguous keyword(s) redirect to the disambiguation article.
+
+## HTML
+
+HTML must not be used, with exception for [comments](#comments). The structure of the article must be redone if HTML is used.
+
+### Comments
+
+HTML comments should be used for marking to-dos, but may also be used to annotate text. They should be on their own line, but can be placed inline in a paragraph. If placed inline, the start of the comment must not have a space.
+
+Bad example:
+
+```markdown
+HTML comments <!-- TODO explain HTML comments --> should be used for marking to-dos or annotate text.
+```
+
+Good example:
+
+```markdown
+HTML comments<!-- TODO explain HTML comments --> should be used for marking to-dos or annotate text.
+```
+
+## Editing
+
+### End of line sequence
+
+*Caution: Uploading Markdown files using `CRLF` (carriage return and line feed) via GitHub will result in those files using `CRLF`. To prevent this, set the line ending to `LF` (line feed) before uploading.*
+
+Markdown files must be checked in using the `LF` end of line sequence.
+
+### Escaping
+
+*Caution: Titles are parsed as plain text and must not be escaped.*
```

Comment:
this probably should be the second sentence, otherwise it makes it look like both sentences are about titles (probably due to short sentences)